node/network/approval-distribution/src/lib.rs

```diff
@@ -244,10 +244,13 @@ impl State {
 						candidates,
 					});
 					new_hashes.insert(meta.hash.clone());
+
+					// In case there are duplicates, we should only set this if the entry
+					// was vacant.
+					self.blocks_by_number.entry(meta.number).or_default().push(meta.hash);
```

Comment:
I believe doesn't change the logic? Some random style changes?

Reply:
This was previously outside of the `match map.entry`, now it's inside the `Occupied` branch

Reply:
emm, it's inside of the `Vacant` branch and we have continue for `Occupied`, so this shouldn't change the logic, right?

Reply:
Ah, you're right

Reply:
I find the current change a bit more readable so I'd like to keep it, if you have no objection.

Reply:
yeah, just wanted to confirm